.github/workflows/release.yaml

```diff
@@ -35,14 +43,15 @@ jobs:
         run: make updateversion VERSION=${{env.VERSION}}
       - name: Create PR
         id: cpr
-        uses: peter-evans/create-pull-request@ad43dccb4d726ca8514126628bec209b8354b6dd
+        uses: peter-evans/create-pull-request@331d02c7e2104af23ad5974d4d5cbc58a3e6dc77
         with:
           add-paths: .
-          commit-message: "Update version to ${{env.VERSION}}"
-          branch: release/${{env.VERSION}}
+          commit-message: "Update version to v${{env.VERSION}}"
+          branch: release/v${{env.VERSION}}
           delete-branch: true
-          title: "Release ${{env.VERSION}}"
-          body: Release prepared for ${{env.VERSION}}
+          title: "Release v${{env.VERSION}}"
```

Comment:
won't these `v`s be doubled? (I realize this looks a lot like https://github.com/bufbuild/buf/blob/main/.github/workflows/release.yaml so maybe not?)

Reply:
these will not, the input should be x.y.z without any v, even though that is not enforced at this stage 😂